[EASY] Added transaction hash logging

mstrug opened the pull request:
# Description
Extracted transaction hash from on-chain event metadata and added it to the log message.
Adding transaction hash to the logs will let anyone with the access to the logs easily find order id and quote from the transaction hash.

# How to test
Execute test trade on staging and observe the logs.

squadgazzz commented:
Could you provide a better description, especially explaining why we need this? From the updated log it is not clear what exactly the TX hash is mentioned.

MartinquaXD reviewed (approved):
Without refactoring the code significantly this is probably the least invasive way to add this log info. 👌

mstrug commented:
> Could you provide a better description, especially explaining why we need this? From the updated log it is not clear what exactly the TX hash is mentioned.

I've updated PR description and changed log field name to: `onchain_transaction_hash` so I think now it is more self descriptive.
